auth_password_settings/changelog

```diff
@@ -0,0 +1,6 @@
+2015-02-24: 1.0
```

Comment:
Do you mean, I put changelog in README and remove changelog file ?

Please correct me

Thanks,

Dhaval Patel
Team Lead Engineer
Denero Team.
www.deneroteam.com
Skype : deneroteam
Mobile : +91-9327694716


On Thu, Feb 26, 2015 at 3:06 PM, Pedro M. Baeza <notifications@github.com>
wrote:

> In auth_password_settings/changelog
> <https://github.com/OCA/server-tools/pull/134#discussion_r25414288>:
>
> > @@ -0,0 +1,6 @@
> > +2015-02-24: 1.0
>
> This can be part of the README file.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OCA/server-tools/pull/134/files#r25414288>.
>